app/src/main/res/values/strings.xml

```diff
@@ -326,4 +327,12 @@
     <string name="alarm_backup">Alternative Alarm</string>
     <string name="alarm_fallback_info">System alarm adopted as selected radio station failed to load</string>
     <string name="alarm_back_desc">Alternative system alarm adopted when the radio station fails to load</string>
+
+    <string name="context_menu_share">Share</string>
```

Comment:
Since these strings are copy-paste of already existing ones, I think we should have references here, e.g.
```
<string name="context_menu_share">@string/detail_share</string>
```

Reply:
Oh yes, absolutely!